.gitignore

```diff
@@ -19,3 +19,6 @@ yarn-error.log*
 *.njsproj
 *.sln
 *.sw*
+
+# Coverage Reports
+coverage
```

Comment:
Hey, thanks a lot for the contribution, but a new line is missing here. Could you add it here or install the `editorconfig` plugin for your editor as described here (https://editorconfig.org/#download). Then this should happen automatically.

I also created an issue about this here: https://github.com/gothinkster/vue-realworld-example-app/issues/226

Reply:
fixed the issue